tests/server/test_compilerservice.py

```diff
@@ -2016,3 +2023,84 @@ async def test_overlapping_env_vars(mocked_compiler_service, server, client, env
             env_vars={"var": "val", "test": "123"},
             mergeable_env_vars={"var": "otherval", "somekey": "someval"},
         )
+
+
+class Dudreport:
```

Comment:
What does `Dud` mean?

Reply:
I assumed it to be a common word, perhaps it is not. 
(such a nice word though) 

I'll change it to mock. 

![image](https://github.com/user-attachments/assets/305444e5-e38d-4aab-b425-5b0ebc4a3305)